andela/pippin-ah-frontend
# [Chore : # 162727507] : Setup sass and css

#### What does this pull request do?
- Setup  Sass and Css

#### Description of task to be completed?
- [x] Install node-sass
- [x] install css-loader
- [x] install  sass-loader
- [x] install style-loader
- [x] include loaders in wepack.common.js
- [x] create style folder and a sample index.css file to test

#### How should this be tested?
- On the terminal, run `npm start:dev` 
- On the page, you should see a horse blood color for the header

#### What are the relevant pivotal tracker stories?
[#162727506](https://www.pivotaltracker.com/story/show/162727506)
